src/summary/history/notes-card.test.tsx

```diff
@@ -21,34 +28,58 @@ describe("<NotesCard/>", () => {
   });
 
   beforeEach(mockFetchPatientEncounters.mockReset);
+  beforeEach(mockUseCurrentPatient.mockReset);
 
-  it("renders successfully", () => {
+  it("renders successfully", async () => {
     mockFetchPatientEncounters.mockReturnValue(
       Promise.resolve(mockPatientEncounters)
     );
-    render(
-      <BrowserRouter>
-        <NotesCard match={match}></NotesCard>
-      </BrowserRouter>
-    );
+    mockUseCurrentPatient.mockReturnValue([
+      false,
+      mockPatient,
+      mockPatient.id,
+      null
+    ]);
+    await wait(() => {
```

Comment:
Shouldn't it pass even without `wait` here?

Reply:
fixed this issue